tests/runner/src/util.rs

```diff
@@ -72,46 +73,95 @@ macro_rules! build_nullable_iter {
     };
 }
 
+/// # Notice
+///
+/// Caller should guarantee the `null_mask` has enough length.
 pub fn values_to_string(
     data_type: ColumnDataType,
     values: Values,
     null_mask: Vec<u8>,
+    row_count: usize,
 ) -> Vec<String> {
-    let bit_vec = BitVec::from_vec(null_mask);
+    let mut bit_vec = BitVec::from_vec(null_mask);
+    // Assume the input data is valid.
+    unsafe {
+        bit_vec.set_len(row_count);
```

Comment:
What about using [truncate()](https://docs.rs/bitvec/latest/bitvec/vec/struct.BitVec.html#method.truncate)? Then we could avoid creating an invalid `bitvec` even if `row_count` is incorrect.

Reply:
They are different. Truncate can only shorten the vec, but I want to set its len precisely the same as row_count. Otherwise some elements will disappear because the "inferred" length will not count tailing 0.

Reply:
Could we add an assertion like `assert!(bit_vec.capacity() >= row_count)` here? Then the test runner could avoid out-of-bound memory access if there are some bugs in the DB.

Reply:
Or could we just call [resize(row_count, false)](https://docs.rs/bitvec/latest/bitvec/vec/struct.BitVec.html#method.resize)?

Reply:
> Or could we just call [resize(row_count, false)](https://docs.rs/bitvec/latest/bitvec/vec/struct.BitVec.html#method.resize)?

That's what I want 👍